clang/lib/CIR/Dialect/IR/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ add_clang_library(MLIRCIR
   CIRMemorySlot.cpp
   CIRTypes.cpp
   CIRDataLayout.cpp
+  FPEnv.cpp
```

Comment:
This shouldn't be here. Since you removed the file, this is causing your CI build to fail.